ui/src/components/navbar.tsx

```
@@ -0,0 +1,163 @@
+  return (
+    <Link
+      href={href}
+      className="flex items-center text-gray-300 transition-colors hover:text-red-400"
+    >
+      <span className="mr-2">{icon}</span>
+      <span>{label}</span>
+    </Link>
+  );
+}
+
+interface User {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)